rustls/src/msgs/deframer.rs

```diff
@@ -335,7 +335,7 @@ impl MessageDeframer {
 }
 
 #[derive(Default)]
-pub struct DeframerBuffer {
+pub struct DeframerVecBuffer {
```

Comment:
If we're going to rename this, let's just introduce it under this name in the first place.